src/components/ui.js

```
@@ -16,7 +74,12 @@ export function Base({
 }
 
 export function Container({ width = "normal", ...props }) {
-  return <Base cx={[styles.containers[width]]} {...props} />
+  return (
+    <Base
+      cx={[safeStyles.containers && safeStyles.containers[width]]}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)